imports/plugins/included/ui-search/client/templates/productSearch/productResults.js

```diff
@@ -56,6 +57,31 @@ Template.productResults.events({
     event.preventDefault();
     loadMoreProducts();
   },
+  "change [data-event-action=filterSearch]": (event) => {
+    // {vendor: [China, ireland]}
+    const key = event.target.parentNode.id;
+    const value = event.target.value;
+
+    this.filterItems.forEach((item) => {
+      let itemKey = Object.keys(item).toString();
+      if (itemKey === key) {
+        if (key === "vendor" && !item[itemKey].includes(value)) {
+          item[itemKey].push(value);
+        } else if (key === "vendor") {
+          item[itemKey].splice(item[itemKey].indexOf(value), 1);
+        } else {
+          console.log("item", item["0"]);
+            console.log("key", key);
+          item[itemKey] = value;
+        }
+      }
+    });
+    console.log(this.filterItems);
+    console.log(this.getProducts());
+    // console.log(ProductSearch.find().fetch());
+    // console.log(event.target.value);
+    // console.log(event.target.parentNode.id);
```

Comment:
Remove lines 81 - 83. Avoid leaving extraneous lines of code in your work when done developing and test before pushing to git.